tests/unit/jobs/ContentJobsTest.php

```diff
@@ -737,4 +737,25 @@ public function itShouldForceDeleteOneContent()
         $this->assertNull(Content::find($first->id));
         $this->assertNotNull(Content::withTrashed()->find($second->id));
     }
+
+
+    /** @test */
+    public function cantUpdateContentWithNonExistingThumb()
+    {
+        $content = $this->tester->haveContent([
+            'type' => 'content'
+        ]);
+
+
+        try {
+            dispatch_now(new UpdateContent($content, [
+                'thumb_id' => nonExistingThumbId
```

Comment:
It's better to have this hardcoded here - it makes the test more readable at first glance, you don't need to search this variable in the different file.

So having some variable in a different place it overcomplication.